browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -819,6 +825,19 @@ until they verify, or until 90 days have passed.
       <message name="IDS_BRAVE_SYNC_V2_MIGRATE_INFOBAR_COMMAND" desc="Command text shown to users who used a previous version of Brave Sync and need to perform setup again for the new version.">
         Begin Setup
       </message>
+      <!-- P3A Android Infobar -->
+      <message name="IDS_BRAVE_CONFIRM_P3A_INFO_BAR" desc="Text that shows on the info bar where user acknowledges the P3A message">
```

Comment:
`INFO_BAR` -> `INFOBAR` ?

Reply:
thanks, this must be `INFOBAR` indeed

Reply:
this is fixed at https://github.com/brave/brave-core/pull/7016/commits/dd980e5648f170f0e3ca3cc827edb16ee7b6e529#diff-610aa766aabf8f9f7575fdf7e6c497348a13788e0e1bc2e98868ecc115cedba0R832